test/index.spec.js

```diff
@@ -97,4 +97,17 @@ describe('freeway', () => {
     const output = new Uint8Array(await res.arrayBuffer())
     assert.doesNotReject(CarReader.fromBytes(output))
   })
+
+  it('should use a rollup index', async () => {
+    const input = [{ path: 'sargo.tar.xz', content: randomBytes(609261780) }]
+    const { dataCid, carCids } = await builder.add(input)
+
+    await builder.rollup(dataCid, carCids)
```

Comment:
looking into test `should get a big file` without the rollup, it is exactly the same, which means we are not validating it comes from rollup? 

could we wire a call count or something like that that we could try to leverage here?

Reply:
I was thinking to remove the non-rollup indexes, so that the only way it can successfully be served would be via rollup.